### DISCUSSION TITLE
Log in with Passkeys

### DISCUSSION THREAD

**Discussion by rushout912** [dani-garcia/vaultwarden#4414] [General]
I just wanted to ask if there is an ETA for this feature update, which is currently in beta on Bitwarden, for Vaultwarden? I can create a post on Vaultwarden Forum's feature request section if discussions isn't a right place to ask, thanks.

[Log in with Passkeys beta](https://bitwarden.com/help/login-with-passkeys/)

**Comment by BlackDex** [comment]
Short answer: **No**

Long answer: This project does not have ETA's on features, or a release schedule or milestones or something similar like that.
If a feature is going to be added there will be a PR for it, and you can watch that PR to see the status of it.

**Reply by ptman** [reply]
I agree on not having a ETA. But is there a roadmap / backlog / prioritized list?

**Reply by BlackDex** [reply]
Something which comes close is this #246

**Comment by yeahwong** [comment]
I think this is a great feature.
